modules/ROOT/pages/B2B-overview.adoc

```diff
@@ -47,9 +49,10 @@ and any associated validations that require an acknowledgment (such as TA1, 997,
 ** Message types to use. For example, a purchase order in X12 is an 850 message.
 You also need to know the X12 version to use in your transactions (such as 004010, 005010, and so on).
 
-** Your partner's digital certificate.
+** Your partners' digital certificates.
+
+Now you are ready to configure yourself as a Host.
 
 == See Also
```

Comment:
Consider not limiting See Also content to the "next" chronological topic.
For example, related to my suggestion that we not detail the Endpoints bullet list item here, because it has its own topic, that might qualify as a See Also item if you feel that users will be stumped by its brevity.
But See Also is pretty subjective and relies on your estimation of your audience's needs and patience level....